data/mods/Magiclysm/worldgen/orc_village.json

```diff
@@ -0,0 +1,40 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "object": {
+      "fill_ter": "t_dirt",
+      "rows": [
+  "111111111111111111111111111111111111111111111111111111111",
+  "1.......................................................1",
+  "1.......................................................3",
+  "1.......................................................1",
+  "1.......................................................3",
+  "1.......................................................1",
+  "1.......................................................1",
+  "1.......................................................1",
+  "1.......................................................1",
+  "1.......................................................1",
+  "1.......................................................1",
+  "1.......................................................1",
+  "1.......................................................1",
+  "111111111111111111111111111111111111111111111111111111111",
+  "                                                         " ]
+        },
+        "place_monsters": [
+  { "monster": "GROUP_ORC", "x": [ 5, 5 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] },
+  { "monster": "GROUP_ORC", "x": [ 5, 45 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] },
+  { "monster": "GROUP_ORC", "x": [ 5, 45 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] },
+  { "monster": "GROUP_ORC", "x": [ 5, 45 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] }
+  ],
+  {
+    "type": "palette",
+    "id": "orc_palette",
+    "terrain": {
+      "3": "t_door_makeshift_c",
+      "1": "t_wall_rammed_earth",
+      ".": "t_dirtfloor"
+    }
+   }
+  }
+]
```

Comment:
```suggestion
[
  {
    "type": "palette",
    "id": "orc_palette",
    "terrain": { "3": "t_door_makeshift_c", "1": "t_wall_rammed_earth", ".": "t_dirtfloor" }
  },
  {
    "type": "mapgen",
    "method": "json",
    "object": {
      "fill_ter": "t_dirt",
      "rows": [
        "111111111111111111111111111111111111111111111111111111111",
        "1.......................................................1",
        "1.......................................................3",
        "1.......................................................1",
        "1.......................................................3",
        "1.......................................................1",
        "1.......................................................1",
        "1.......................................................1",
        "1.......................................................1",
        "1.......................................................1",
        "1.......................................................1",
        "1.......................................................1",
        "1.......................................................1",
        "111111111111111111111111111111111111111111111111111111111"
      ],
      "palettes": [ "orc_palette" ],
      "place_monsters": [
        { "monster": "GROUP_ORC", "x": [ 5, 5 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] },
        { "monster": "GROUP_ORC", "x": [ 5, 45 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] },
        { "monster": "GROUP_ORC", "x": [ 5, 45 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] },
        { "monster": "GROUP_ORC", "x": [ 5, 45 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] }
      ]
    }
  }
]
```
This should pass the linter with no problems, unless something got messed up in copy/pasting - even then, it should only be spaces that could need changing.